Products/DataCollector/SnmpClient.py

```diff
@@ -244,7 +244,10 @@ def clientFinished(self, result):
                 log.exception("Device %s had an error: %s",self.hostname,result)
         else:
             self._sendStatusEvent('SNMP agent up', eventKey='agent_down', severity=Event.Clear)
-        self.proxy.close()
+        try:
+            self.proxy.close()
+        except AttributeError:
+            log.info("Caught AttributeError closing SNMP connection.")
```

Comment:
I think I originally had that as a warning, and someone (JP?) had me change it to info.